File: samples/java-only/src/test/java/example/MyAppTest.java
package example;

import static org.junit.Assert.assertTrue;

import org.junit.Test;

public class MyAppTest {

    @Test
    public void testThatAspectIsCalled() {
        MyApp.sayHi();
        assertTrue("The aspect was not called", StaticStateVariable.wasAspectCalled);
    }
}

File: samples/kotlin/src/test/kotlin/example/MyAppTest.kt
package example

import junit.framework.Assert.assertTrue
import org.junit.Test

class MyAppTest {
    @Test
    fun `test that aspect is called`() {
        MyApp.sayHi()
        assertTrue(StaticStateVariable.wasSayHiAspectCalled)
    }

    @Test
    fun `test that return nothing aspect is called`() {
        MyApp.returnNothing()
        assertTrue(StaticStateVariable.wasReturnNothingAspectCalled)
    }
}